terraso_backend/apps/export/handlers.py

```diff
@@ -0,0 +1,40 @@
+# Copyright © 2021-2025 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+from corsheaders.signals import check_request_enabled
+
+
+def cors_allow_export_urls(sender, request, **kwargs):
```

Comment:
question: how are these URLs intended to be called? i thought CORS was only needed so that JavaScript loaded from some other domain could make a request to an endpoint on your domain, but that it wasn't needed if the user is just opening the URL directly, but maybe i'm misunderstanding